cmake/Options/TileDBAssertions.cmake

```diff
@@ -8,7 +8,7 @@ if(TILEDB_ASSERTIONS)
   endif()
   # On non-Debug builds cmake automatically defines NDEBUG, so we
   # explicitly undefine it:
-  if( NOT uppercase_CMAKE_BUILD_TYPE STREQUAL "DEBUG" )
+  if( NOT CMAKE_BUILD_TYPE STREQUAL "DEBUG" )
```

Comment:
With that change, shouldn't the literal now be 'Debug'?

Reply:
Done, thanks.